templates/catppuccin-flavor.tmuxtheme.tera

```diff
@@ -0,0 +1,26 @@
+---
+whiskers:
+  version: "2.1.0"
+  matrix:
+    - flavor
+  filename: "themes/{{flavor.identifier}}/{{flavor.identifier}}.tmuxtheme"
```

Comment:
Wouldn't it make more sense to have a directory structure like:

```
themes/
├─ catppuccin_latte.tmuxtheme
├─ catppuccin_frappe.tmuxtheme
├─ catppuccin_macchiato.tmuxtheme
├─ catppuccin_mocha.tmuxtheme
```

Feels like the flatter version is nicer, is there a practical reason for nesting it further?

Reply:
Was from when the matrix included accents